### PR TITLE
Design 1

### PR DESCRIPTION
**Pull request by Sn8z**
Implements a basic layout and search funtionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1 @@
 # RottenTomato
```
Comment (Snajtz) [inline README.md:1]: You might want to have something like : in progress

**Comment by Snajtz** [comment]
I think this is ok to merge!

File: app/views/start/index.html.erb [hunk 2]
```diff
@@ -1 +1,52 @@
-hej
\ No newline at end of file
+<!-- Header -->
+<div id="header">Header</div>
```
Comment (Snajtz) [inline app/views/start/index.html.erb:2]: Probably should add something more here, maybe just some kind of single-color navbar

**Comment by Snajtz** [comment]
The search "button" doesn't work, it changes apperance on click but doesn't do anything. What is it intended to do?

**Comment by Brinkentosh** [comment]
Looks good!

**Pull request merged**